Makefile

```diff
@@ -3,6 +3,7 @@ all: check test race
 .PHONY: check
 check:
 	go build ./roc
+	go test ./roc -run xxx
```

Comment:
just curious: we don't have a test that's called `xxx` - does this code checks (compiles) test files?

Reply:
Yep. `-run xxx` is just a convention to say "compile and run nothing", though maybe not a widely used one. I added this check because otherwise if test are not compiling, the linter in `make check` will produce cryptic errors.